.github/workflows/python-cleanliness.yml

```diff
@@ -16,10 +16,15 @@ jobs:
         with:
           submodules: 'recursive'
 
+      # Run flake8 but exclude third-party libraries
+      - run: pipx run flake8 --exclude=MAVProxy/modules/mavproxy_cesium,MAVProxy/modules/lib
+      # Once this has been running for a while, we can:
+      # - delete the lines below
+      # - delete scripts/run_flake8.py
+      # - remove all instances of AP_FLAKE8_CLEAN (git grep AP_FLAKE8_CLEAN)
+
       - name: Install dependencies
-        run: |
-          python -m pip install -U flake8
+        run: python -m pip install --upgrade flake8
```

Comment:
Please revert these patches.  They are unrelated to the topic of the PR and don't improve things.